File: SoloLearn/Popsicles.py
siblings = int(input('So kem cho 1 group :').strip())
popsicles=int(input('Tong so kem : ').strip())
pop_num=popsicles//siblings # numbers of popsicle for 3 group = siblings
left=popsicles % siblings
print('give away' if pop_num >=2 and left >= 0 else 'eat them yourself')

File: Java Code/Popsicles.java
import java.util.Scanner;

public class Popsicles {

	public static void main(String[] args) {
		String ans="";
		Scanner sc = new Scanner(System.in);
		System.out.print("Siblings Numbers: ");
		int siblings =sc.nextInt();
		System.out.print("Popsicles Numbers: ");
		int popsicles=sc.nextInt();
		sc.close();
		int left = popsicles % siblings;
		int groups=popsicles /siblings;
		if (left >=0 && groups>=2) {
			ans="give away";
		} else {
			ans="eat them yourself";
		}
		System.out.println(ans);
	}

}
